src/main/resources/frontend/src/services/submissionService.ts

```diff
@@ -50,4 +50,21 @@ export const submitGet = async (): Promise<boolean> => {
     const body = await response.json() as SubmitGetResponse;
 
     return body.inQueue;
+}
+
+export const reRunSubmissionsPost = async () => {
+    const response = await fetch(useAppConfigStore().backendUrl + "/api/admin/submissions/rerun", {
```

Comment:
@DallinFromEarth @pawlh  Are we concerned about validating the permissions of whomever is sending the request? Currently, this would accept a rerunsubmission from anyone without respect to their logged-in status or otherwise.